src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -339,4 +340,21 @@ default GameModeData getGameModeData() {
      */
     void setSpectatorTarget(@Nullable Entity entity);
 
+    /**
+     * Gets the {@link WorldBorder} for this player. If no border is set, a
+     * border that is the same as the {@link World}'s world border, in terms of 
+     * properties, is returned.
+     *
+     * @return The {@code WorldBorder} of this player.
+     */
+    WorldBorder getWorldBorder();
+
+    /**
+     * Sets the {@link WorldBorder} instance for this player to the given world 
+     * border.
+     * 
+     * @param border The world border to be used.
+     * @param cause The cause of the border's change.
+     */
+    void setWorldBorder(WorldBorder border, Cause cause);
```

Comment:
You mean like a method that set the player's world border to that of their world's?